docs/modules/chains/getting_started.ipynb

```diff
@@ -40,6 +40,8 @@
    "source": [
     "from langchain.prompts import PromptTemplate\n",
     "from langchain.llms import OpenAI\n",
+    "import os\n",
+    "os.environ[\"OPENAI_API_KEY\"] = \"Your-API-KEY\"\n",
```

Comment:
Given that the API key is not set to a correctly value here, would you mind commenting this code out. It will make it easy for us to keep on re-running the notebook if we need to update it while also informing users that they can set the value dynamically (but first have to uncomment the code)

Reply:
Thank you for your advice. That's right. I agree with you.
So, I deleted two lines and committed again with the message: "Deleted key setting lines because how to set the OpenAI key has already been guided." I still need to familiarize myself with open-source contributions. So, if I still need to correct something, please let me know.